docs/docs.json

```diff
@@ -396,10 +388,7 @@
           {
             "group": "Policies",
             "icon": "user-police-tie",
-            "pages": [
-              "studio/policies",
-              "studio/lint-policy/linter-rules"
-            ]
+            "pages": ["studio/policies", "studio/lint-policy/linter-rules"]
```

Comment:
Please polish update the overview https://wundergraphinc-jesse-grpc-connect-docs.mintlify.app/tutorial